gradle.properties

```diff
@@ -1,4 +1,5 @@
 profiler.version=0.22.0-SNAPSHOT
+org.gradle.jvmargs=-Xmx3100m -XX:MaxMetaspaceSize=768m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
```

Comment:
💭 Added since otherwise I got errors on CI that GC is trashing

Reply:
`3100m` seems oddly specific. How about `4g`?

Reply:
We can go with 4g

Reply:
Done